tasks/request_files/request_files.py

```diff
@@ -579,9 +567,10 @@ def restore_object(
             f"{c_err}. bucket: {db_glacier_bucket_key} file: {key} Job ID: {job_id}"
         )
         raise c_err
-    if restore_result["ResponseMetadata"]["HTTPStatusCode"] == 202:
+    if restore_result["ResponseMetadata"]["HTTPStatusCode"] == 200:
         # Will have a non-error path after https://bugs.earthdata.nasa.gov/browse/ORCA-336
-        raise KeyError(f"File with key {key} in bucket {db_glacier_bucket_key} has already been restored.")
+        raise ClientError({"key": key, "bucket": db_glacier_bucket_key},
```

Comment:
I do not believe the inputs are correct here. If you run this the resulting message is : `An error occurred (Unknown) when calling the File '/some/fake/key/file-goes-here.txt' in bucket 'my-orca-bucket' has already been restored. operation: Unknown`

Looking at the code for `ClientError` here https://github.com/boto/botocore/blob/develop/botocore/exceptions.py I believe you want the following:
```
raise ClientError({"Error": {"Code": "HTTPStatus: 200", "Message": f"File '{key}' in bucket '{db_glacier_bucket_key}' has already been recovered."}}, "restore_object")
```

If you run a quick test with that you will get the following error message coming out for logging

```
>>> try:
...     key = "/some/fake/key/file-goes-here.txt"
...     db_glacier_bucket_key = "my-orca-bucket"
...     raise ClientError({"Error": {"Code": "HTTPStatus: 200", "Message": f"File '{key}' in bucket '{db_glacier_bucket_key}' has already been recovered."}}, "restore_object")
... except ClientError as e:
...     print(e)
...
An error occurred (HTTPStatus: 200) when calling the restore_object operation: File '/some/fake/key/file-goes-here.txt' in bucket 'my-orca-bucket' has already been recovered.
```

Reply:
Serves me right for not checking the output string. Fixed.